pkgs/development/interpreters/python/default.nix

```diff
@@ -61,7 +61,7 @@
             selfTargetTarget = pythonOnTargetForTarget.pkgs or {}; # There is no Python TargetTarget.
           };
           hooks = import ./hooks/default.nix;
-          keep = lib.extends hooks pythonPackagesFun;
+          keep = self: { inherit (self) wrapPython; };
```

Comment:
would be idea to splice wrapPython too so i could move forward with https://github.com/NixOS/nixpkgs/pull/204387

Reply:
done